pkg/services/rpcsrv/server.go

```diff
@@ -1758,14 +1757,14 @@ func (s *Server) getrawtransaction(reqParams params.Params) (any, *neorpc.Error)
 		_header := s.chain.GetHeaderHash(height)
 		header, err := s.chain.GetHeader(_header)
 		if err != nil {
-			return nil, neorpc.NewRPCError("Failed to get header for the transaction", err.Error())
+			return nil, neorpc.NewInvalidParamsError(fmt.Sprintf("Failed to get header for the transaction: %s", err.Error()))
```

Comment:
Let's replace it with `InternalServerError`, something is wrong with the DB, because we have `height` returned from `GetTransaction`.

Reply:
replaced